lib/ts/recipe/thirdparty/api/implementation.ts

```diff
@@ -22,52 +27,42 @@ export default function getAPIInterface(): APIInterface {
         },
 
```

Comment:
we remove the stuff to do with factor id is completed. Keep the first factor check, but remove everything else related to MFA here.

Reply:
Done